deployment/gcp/sql.tf

```diff
@@ -20,7 +20,7 @@ resource "google_sql_database_instance" "ticket_booth" {
       value = "on"
     }
 
-    deletion_protection_enabled = true
+    # deletion_protection_enabled = true
```

Comment:
Why are we taking this off?

Reply:
so that I can destroy and recreate until we get the instance name right

Reply:
I will make sure this is turned on once we are confirmed with the correct naming